docs/intro.md

```diff
@@ -4,4 +4,16 @@ sidebar_position: 1
 
 # Intro
 
-A hacker's guide to hackathons, made by the HackUTD Team.
+Most hackathon projects will want some sort of user interface - after all, you want the judges to be able to visualize your idea, right? Many projects also contain some sort of system that deals with data, whether that be user data or data from external sources. This guide explains and provides resource to learn about user interfaces and data driven systems.
+
+## The User Interface (Frontend)
+
+User interfaces and the software behind them are also known as the **frontend**. Frontends are what users see and interact with. When you go to a website and log in, you're interacting with the **frontend** of that website. The term **frontend** describes the text, buttons, links, input fields, etc. on a webpage and the software that goes into creating and composing them.
+
```

Comment:
You can prob also include what languages are included in frontend so the reader can make more connections to what is associated in frontend. So you can add "Technologies include HTML, CSS, JS and nowadays React, Tailwind, Material UI" or whatever u wanna say.